app/components/jobseekers/account_survey_link_component/account_survey_link_component.html.slim

```diff
@@ -1,3 +1,5 @@
 .account-survey-cta
-  .account-survey-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="account-survey-cta"
-    = t(".survey_text_html", link: link_to_survey)
+  .wrapper
+    .account-survey-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="account-survey-cta"
```

Comment:
is the id on this needed? although see you havent added it in this PR

Reply:
Yeah I don't know.

Reply:
i guess leave it, its not your change